aws/resource_registry.go

```diff
@@ -64,7 +64,6 @@ func getRegisteredRegionalResources() []AwsResource {
 		&resources.AppRunnerService{},
 		&resources.BackupVault{},
 		&resources.ManagedPrometheus{},
-		&resources.Grafana{},
```

Comment:
can you rebase? both **Grafana** and **ElasticCacheServerless** where already merge